pytest.ini

```diff
@@ -7,3 +7,8 @@ env_files =
 testpaths =
     tests/unit
     tests/functional
+markers =
+    delta
```

Comment:
Please do document the newly introduced markers

https://docs.pytest.org/en/7.1.x/example/markers.html

Reply:
https://docs.pytest.org/en/7.1.x/example/markers.html#registering-markers